app/src/main/java/com/tvtracker/MainActivity.kt

```diff
@@ -63,12 +63,13 @@ import com.google.firebase.auth.FirebaseAuth
 import com.tvtracker.dto.User
 import androidx.compose.material.darkColors
 import androidx.compose.material.lightColors
+import androidx.compose.ui.res.stringResource
 
 
 class MainActivity : ComponentActivity() {
 
     private var openDialog by mutableStateOf(false)
-    private val viewModel: BrowseViewModel by viewModel<BrowseViewModel>()
+    private val viewModel: BrowseViewModel by viewModel()
```

Comment:
Was not necessary to add <BrowsViewModel>

Reply:
Ehhhh, I don't know that it's hurting anything. Either way works.